src/components/pages/user-registration-page/user-registration-page.styles.ts

```diff
@@ -59,6 +59,11 @@ export default makeStyles((theme: Theme) => createStyles({
 
     '& *:nth-child(1)': theme.typography.h1,
     '& *:nth-child(2)': theme.typography.subtitle1,
+    '& a:nth-child(1)': {
+      ...theme.typography.subtitle1,
+      fontWeight: theme.typography.fontWeightBold,
+
+    },
```

Comment:
It looks like the existing styles here weren't quite specific enough, which is why that link was requiring a special override to control its size. Doing this instead should generalize a bit better...
```suggestion
    '& > *:nth-child(1)': theme.typography.h1,
    '& > *:nth-child(2)': theme.typography.subtitle1,
    '& a': {
      fontWeight: theme.typography.fontWeightBold,
    },
```